jobs/build-podman.Jenkinsfile

```diff
@@ -0,0 +1,339 @@
+def gitref, commit, shortcommit
+
+def artifacts = [
+    [platform: "qemu", suffix: "qcow2"],
+    [platform: "applehv", suffix: "raw"],
+    [platform: "hyperv", suffix: "vhdx"]
+]
+
+node {
+    checkout scm
+    // these are script global vars
+    pipeutils = load("utils.groovy")
+}
+
+// Keep it here for now instead of utils
+// since it is podman job only
+
+def generate_disk_images(artifacts, shortcommit, arch, repo) {
+    generate_diskvar_json(shortcommit, arch, artifacts, repo)
+    // First create the qcow2 and store it in cache
+    supermin_run("--cache", arch, "qcow2", shortcommit)
+    def artifacts_runvm = artifacts.drop(1) // drop qcow2
+    parallel artifacts_runvm.collectEntries{artifact -> [artifact["platform"], {
+        supermin_run("--snapshot", arch, "${artifact["suffix"]}", shortcommit)
+    }]}
```

Comment:
ouch.. this isn't the most ergonomic because of the way runvm-osbuild essentially takes one platform/filename at a time. I'll maybe see if there is way to make it better so we can just create all the platforms with one single run.